test/integration/test/captions.test.js

```diff
@@ -0,0 +1,28 @@
+const clientHelper = require('../src/client-helper');
+const configHelper = require('../src/config-helper');
+
+test('Can get srt captions', async (done) => {
+    const client = clientHelper.getClient(configHelper.getApiKey());
+    const jobList = await client.getListOfJobs();
+    const jobId = getTranscribedJobId(jobList);
+    expect(jobId).toBeDefined();
+    const captionsStream = await client.getCaptions(jobId);
+    var streamString = '';
+    captionsStream.on('data', data => {
+        streamString += data.toString();
+    })
+    captionsStream.on('end', () => {
+        expect(streamString.length).toBeGreaterThan(0)
+        done();
+    })
+})
+
+function getTranscribedJobId(jobList) {
```

Comment:
This function definition seems to be repeated across multiple files. I would suggest creating either a helper.

Reply:
consider defining this in client helpers?